smt/ctx.cpp

```diff
@@ -17,6 +17,7 @@ void context::init() {
   // Disable Z3's use of UFs for NaNs when converting FPs to BVs
   // They generate incorrect formulas when quantifiers are involved
   Z3_global_param_set("rewriter.hi_fp_unspecified", "true");
+  Z3_global_param_set("smt.recfun.depth", "32768");
```

Comment:
seems like a huge value. Why?

Reply:
This was set to the large number to minimize false positive.
If we can use static analysis, it can be the maximum length of string that is given to strlen.

Reply:
actually I am fine with setting it with smaller constants first and adding analysis for it.

Reply:
ok, I suggest getting rid of this change altogether. This parameter is the minimum initial unrolling; Z3 will bump it automatically.
This can be benchmarked afterward.

Reply:
I guess the number of unrolling can be smaller than this parameter; for example, setting this as 5 makes strlen.src.ll fail, but setting this as 6 makes it succeed.
Instead of 32768, I fixed it as 6 to make the strlen.src.ll test succeed - but it may not be sufficient for existing benchmarks.

Reply:
Please ignore my previous comment. 
I agree it seems to be the mininum one. I will remove the change as suggested.